repos/rust-lang/rust-clippy.toml

```
@@ -2,11 +2,16 @@ org = "rust-lang"
 name = "rust-clippy"
 description = "A bunch of lints to catch common mistakes and improve your Rust code. Book: https://doc.rust-lang.org/clippy/"
 homepage = "https://rust-lang.github.io/rust-clippy/"
-bots = ["bors", "rustbot"]
+bots = ["rustbot"]
 
 [access.teams]
 clippy = "write"
 clippy-contributors = "triage"
 
 [[branch-protections]]
 pattern = "master"
+ci-checks = [
+    "conclusion",
+    "conclusion_dev",
+    "conclusion_remark",
+]
```

The Clippy repo needs the same "no approvals required" as miri, so that we can self-merge PRs:

```suggestion
]
required-approvals = 0
```

---

Done :+1: